src/libs/SearchUIUtils.ts

```diff
@@ -933,6 +948,9 @@ function shouldShowYear(
             if (item.submitted && DateUtils.doesDateBelongToAPastYear(item.submitted)) {
                 result.shouldShowYearSubmitted = true;
             }
+            if (item.approved && DateUtils.doesDateBelongToAPastYear(item.approved)) {
+                result.shouldShowYearApproved = true;
+            }
```

Comment:
I missed adding approved to the early exist clause. I'll do that in a follow up PR since I'm touching these lines anyways and the PR is ready for review too https://github.com/Expensify/App/pull/77791